app/etc/di.xml

```diff
@@ -1113,7 +1053,7 @@
     </type>
     <type name="Magento\Framework\Session\Validator">
         <arguments>
-            <argument name="scopeType" xsi:type="const">Magento\Store\Model\ScopeInterface::SCOPE_STORE</argument>
+            <argument name="scopeType" xsi:type="const">Magento\Framework\App\ScopeInterface::SCOPE_DEFAULT</argument>
             <argument name="skippedUserAgentList" xsi:type="array">
```

Comment:
argument `scopeType` should be remove due duplicated in module Store already defined

Reply:
it's overriden in the Magento\Store module, the idea is to separate responsabilities and design scope. app/etc/di.xml should only contain framework di without knowing the context of modules

Reply:
no Magento_Store will override this file